orchagent/portsorch.cpp

```diff
@@ -3513,23 +3526,26 @@ bool PortsOrch::addBridgePort(Port &port)
         return false;
     }
 
-    /* Create a bridge port with admin status set to UP */
-    attr.id = SAI_BRIDGE_PORT_ATTR_ADMIN_STATE;
-    attr.value.booldata = true;
-    attrs.push_back(attr);
-
-    /* And with hardware FDB learning mode set to HW (explicit default value) */
-    attr.id = SAI_BRIDGE_PORT_ATTR_FDB_LEARNING_MODE;
-    auto found = learn_mode_map.find(port.m_learn_mode);
-    if (found == learn_mode_map.end())
-    {
-        attr.value.s32 = SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW;
-    }
-    else
+    if (port.m_type != Port::SYSTEM)
     {
-        attr.value.s32 = found->second;
+        /* Create a bridge port with admin status set to UP */
+        attr.id = SAI_BRIDGE_PORT_ATTR_ADMIN_STATE;
+        attr.value.booldata = true;
+        attrs.push_back(attr);
+
+        /* And with hardware FDB learning mode set to HW (explicit default value) */
+        attr.id = SAI_BRIDGE_PORT_ATTR_FDB_LEARNING_MODE;
+        auto found = learn_mode_map.find(port.m_learn_mode);
+        if (found == learn_mode_map.end())
+        {
```

Comment:
also any reason to skip this for system port ? code here is generic for add systemport as bridge port.
Yes for inband use case this not needed but i feel this code should be kept generic

Reply:
Both local and remote cpu ports need to be added into inband vlan. The port type of remote cpu ports is system port, and thus we have to add the check here to skip the processing that's not required for system port.